src/components/application_manager/src/application_manager_impl.cc

```diff
@@ -3252,9 +2466,12 @@ void ApplicationManagerImpl::UnregisterApplication(
   }
 
 #ifdef SDL_REMOTE_CONTROL
```

Comment:
shall we still have #ifdef SDL_REMOTE_CONTROL here?

Reply:
@yang1070 no, we don't need this ifdefs anymore, I'm working on removing them

Reply:
@yang1070 all redundant ifdefs removed 597cd0d1908d084dc6727034aca63d209987f1f7